packages/fluentui/react-northstar/test/specs/components/Form/FormField-test.tsx

```diff
@@ -72,6 +82,8 @@ describe('FormField', () => {
           },
         }}
       />,
+      // toBeVisible requires that the element be in the document
+      { attachTo: testContainerInfo.testContainer },
```

Comment:
did this break with testing-library upgrade ?

Reply:
Yes, toBeVisible apparently got stricter

Reply:
that's super weird, it's a test that mounts with enzyme but uses custom matcher from testing library .... @layershifter  do we want to do anything about this in N* ?

Not blocking this PR tho

Reply:
We probably want to get rid of Enzyme 🙃
It's unclear when assertions from `jest-dom` are used. For this case it's probably better to replace it, but it's not blocking for this PR.

Reply:
I thought that was weird too but wasn't sure I should migrate the test to fully use testing-library since there's so little usage of testing-library in northstar right now. Open to suggestions (or if either of you want to push an update), or we can just merge it as-is.

Reply:
Let's merge it as it is. Migration to RTL is out of scope of this PR.